docs/containers/debug-node.md

```diff
@@ -124,10 +115,107 @@ Also note that the `debug` logger will not write logs unless enabled via the `DE
 }
 ```
 
-### Automatically launching the browser to the entry page of the application
+### Configuring when the application is "ready"
+
+The extension determines the application is "ready" to receive HTTP connections when it writes a message of the form `Listening on port <number>` to the debug console, as Express.js does by default.  If the application logs a different
+message, then the `pattern` property of the [`dockerServerReadyAction`]((debug-common.md#dockerServerReadyAction-object-properties)) object of the debug launch configuration should be set to a [JavaScript regular expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) that matches that message. The regular expression should include a capture group that corresponds to the port on which the application is listening.
+
+For example, suppose the application logs the following message:
+
+```js
+function onListening() {
+  var addr = server.address();
+  var bind = typeof addr === 'string'
+    ? 'pipe ' + addr
+    : 'port ' + addr.port;
+  debug('Application has started on ' + bind);
+}
+```
+
+The corresponding `pattern` in the debug launch configuration (in `launch.json`) would be:
+
+```json
+{
+    "configurations": [
+        {
+            "name": "Docker Node.js Launch and Attach",
+            "type": "docker",
+            "request": "launch",
+            "preLaunchTask": "docker-run: debug",
+            "platform": "node",
+            "dockerServerReadyAction": {
+                "pattern": "Application has started on port (\\d+)"
```

Comment:
So maybe call out the regular expression `\d+` captures the port number (is one slash an escape char?)

Reply:
Added note to call out the capture group for the port, and the use of the escape character.